src/app/module/feature.ts

```diff
@@ -0,0 +1,6 @@
+export interface IFeature {
+  featureId?: number;
```

Comment:
why do you need ID?

Reply:
In feature if we need to delete only this feature, we can do it with ID. If we do not have such task, I can delete :)